README-MS.md

```diff
@@ -134,11 +134,10 @@ Cari pautan yang mengarahkan secara langsung kepada masalah yang sesuai untuk di
 
 ## Sumbangan inisiatif Sumber Terbuka
 - [Up For Grabs](https://up-for-grabs.net/) - Mengandungi isu projek yang mesra penyumbang baru.
-- [First Timers Only](https://www.firsttimersonly.com/) - Senarai pepijat yang dilabel "first-timer-only".
 - [First Contributions](https://firstcontributions.github.io/) - Buat sumbangan Sumber Terbuka pertama anda dalam 5 minit. Alat dan tutorial untuk membantu pemula memulakan dengan sumbangan. [Here](https://github.com/firstcontributions/first-contributions) adalah kod sumber GitHub untuk laman web dan peluang untuk memberikan sumbangan ke repositori itu sendiri.
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - Program untuk menggalakkan sumbangan Sumber Terbuka. Dapatkan hadiah seperti t-shirt dan pelekat dengan membuat sekurang-kurangnya 4 permintaan tarik pada bulan Oktober.
 - [24 Pull Requests](https://24pullrequests.com) - 24 permintaan tarik('pull request') adalah projek untuk mempromosikan kolaborasi Sumber Terbuka pada bulan Disember.
 
 ## Lesen
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />Kerja ini dilesen dan dibawah hak cipta <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept and added back in.

Reply:
Done e8c131ea1774d90dd3a30cabd529c5f16d1ce230